network-api/networkapi/buyersguide/factory.py

```diff
@@ -17,13 +19,31 @@
 Faker.add_provider(ImageProvider)
 
 
+seed = randint(0, 5000000)
+if settings.RANDOM_SEED is not None:
+    seed = settings.RANDOM_SEED
+
+fake = ValueFaker()
+fake.random.seed(seed)
+
+
 def get_random_category():
     all = BuyersGuideProductCategory.objects.all()
```

Comment:
Out of scope of this PR but it would be nice to rename this variable to something that does not shadow all()

Reply:
good catch, filed https://github.com/mozilla/foundation.mozilla.org/issues/2625